<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetListesContacts</class>
 <widget class="QMainWindow" name="WidgetListesContacts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listes Contact</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Nom"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Entreprise</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Entreprise"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Search">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="ContactNumber">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(65, 65, 65);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre Contact Actuel:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Contact_Results">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>331</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateDebut">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateFin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Search_Date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher Dans Un Intervalle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTrier">
    <property name="title">
     <string>Trier</string>
    </property>
    <addaction name="actionTrier_Par_Ordre_Alphabetique"/>
    <addaction name="actionTrier_Par_Date_De_Creation"/>
   </widget>
   <addaction name="menuTrier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTrier_Par_Ordre_Alphabetique"/>
   <addaction name="actionTrier_Par_Date_De_Creation"/>
  </widget>
  <action name="actionTrier_Par_Ordre_Alphabetique">
   <property name="icon">
    <iconset resource="Ressources_QT.qrc">
     <normaloff>:/rec/Ressources/trier.png</normaloff>:/rec/Ressources/trier.png</iconset>
   </property>
   <property name="text">
    <string>Trier Par Ordre Alphabetique</string>
   </property>
  </action>
  <action name="actionTrier_Par_Date_De_Creation">
   <property name="icon">
    <iconset resource="Ressources_QT.qrc">
     <normaloff>:/rec/Ressources/calendrier.png</normaloff>:/rec/Ressources/calendrier.png</iconset>
   </property>
   <property name="text">
    <string>Trier Par Date De Cr√©ation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Ressources_QT.qrc"/>
 </resources>
 <connections/>
</ui>
